Sources/SentryCrash/Recording/SentryCrash.h

```diff
@@ -207,6 +207,12 @@ static NSString *const SENTRYCRASH_REPORT_ATTACHMENTS_ITEM = @"attachments";
  */
 - (BOOL)install;
 
+/**
+ * Its not really possible to uninstall SentryCrash. Best we can do is to deactivate all the
+ * monitors and clear the `onCrash` callback installed on the global handler.
```

Comment:
I don't know all the specifics of how KSCrash installs its handler, but it should keep a reference to any preexisting handlers, that we could reinstall after removing our own. This probably won't be an issue in practice since most apps probably won't install more than one crash handler, but it's a good thing to keep in mind in case this ever comes up. Some other code might use signals for some other reason.

Reply:
Hmm, isn't that what I'm doing with keeping track of the monitors with `monitoringWhenUninstalled` in `SentryCrash.m`?

Reply:
I believe he is talking about the signal handler pointer. 
It's a valid point, but since we dont do this already I would not include in this PR.